### PR TITLE
Expand {sha} and {version} is user commit message

### PR DESCRIPTION
**Pull request by abitrolly**

### CODE REVIEW

**Comment by waylan** [comment]
Interesting. I didn't even realize this was done with the default message. We should add some documentation that this is supported [here](https://github.com/mkdocs/mkdocs/blob/master/mkdocs/__main__.py#L85).

**Comment by waylan** [comment]
Also, a line should be added to the [release notes](https://github.com/mkdocs/mkdocs/blob/master/docs/about/release-notes.md#development-version)

**Comment by abitrolly** [comment]
Done.

File: mkdocs/__main__.py [hunk 1]
```diff
@@ -83,7 +83,7 @@ def common_options(f):
 no_reload_help = "Disable the live reloading in the development server."
 dirty_reload_help = "Enable the live reloading in the development server, but only re-build files that have changed"
 commit_message_help = ("A commit message to use when commiting to the "
```
Comment (trel) [inline mkdocs/__main__.py]: not part of this original PR, but this should also be 'committing' with two t's

**Pull request merged**